From a66a10e8b9b11ee238a82058bb2bb895b018cb24 Mon Sep 17 00:00:00 2001
From: nSircombe <32057673+nSircombe@users.noreply.github.com>
Date: Fri, 9 Jul 2021 01:02:34 +0100
Subject: [PATCH 2/4] build: fix to include path for ArmPL builds. (#1111)

---
 cmake/blas.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/blas.cmake b/cmake/blas.cmake
index 4a5712496..7dc113fd5 100644
--- a/cmake/blas.cmake
+++ b/cmake/blas.cmake
@@ -1,5 +1,5 @@
 # *******************************************************************************
-# Copyright 2020 Arm Limited and affiliates.
+# Copyright 2020-2021 Arm Limited and affiliates.
 # SPDX-License-Identifier: Apache-2.0
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
@@ -51,6 +51,7 @@ if(DNNL_BLAS_VENDOR STREQUAL "MKL")
 elseif(DNNL_BLAS_VENDOR STREQUAL "OPENBLAS")
     set(BLA_VENDOR "OpenBLAS")
 elseif(DNNL_BLAS_VENDOR STREQUAL "ARMPL")
+    set(CBLAS_HEADERS "armpl.h")
     expect_arch_or_generic("AARCH64")
     if(DNNL_CPU_RUNTIME STREQUAL "OMP")
         set(BLA_VENDOR "Arm_mp")
-- 
2.17.1

